package.json

```diff
@@ -50,6 +53,7 @@
     "@commitlint/config-conventional": "14.1.0",
```

Comment:
somewhere we lost `react-router` dependency. Please add it `npm i react-router --save`. Then check the depcheck with `npm run depcheck`. We however do have the `@types/react-router`.